pkg/extensions/search/common/common_test.go

```diff
@@ -36,6 +36,15 @@ type ImgResponsWithLatestTag struct {
 	Errors               []ErrorGQL           `json:"errors"`
 }
 
+type DetailedRepoInfoResp struct {
```

Comment:
Do you want to keep these as "Detailed"? Maybe change these too?

Reply:
Want to use one keyword, else confusing to code readers

Reply:
Done